hack/local-setup.sh

```diff
@@ -100,16 +101,28 @@ port80=8082
 port443=8445
 
 #1. Create Kind control plane cluster
-kindCreateCluster $KIND_CLUSTER_CONTROL_PLANE $port80 $port443
+kindCreateCluster ${KIND_CLUSTER_CONTROL_PLANE} ${port80} ${port443}
+
 #2. Deploy ingress controller
-deployIngressController $KIND_CLUSTER_CONTROL_PLANE
+deployIngressController ${KIND_CLUSTER_CONTROL_PLANE}
+
 #3. Deploy cert manager
-deployCertManager $KIND_CLUSTER_CONTROL_PLANE
-#4. Deploy argo cd
-deployArgoCD $KIND_CLUSTER_CONTROL_PLANE
+deployCertManager ${KIND_CLUSTER_CONTROL_PLANE}
+
+#4. Deploy external dns
+deployExternalDNS ${KIND_CLUSTER_CONTROL_PLANE}
```

Comment:
@roivaz are you using ExternalDNS? I had removed it as it isn't required.

Reply:
No, I added it back by mistake, was working initially over an older commit and didn't spot the change when I rebased.

Reply:
No worries, can be removed again, just adding unnecessary time to local-setup.